definitions/npm/history_v5.x.x/flow_v0.104.x-/history_v5.x.x.js

```diff
@@ -22,7 +22,7 @@ declare module 'history' {
     go(n: number): void,
     goBack(): void,
     goForward(): void,
-    listen((location: HistoryLocation, action: Action) => void): Unregister,
+    listen(({ location: HistoryLocation, action: Action }) => void): Unregister,
```

Comment:
Does `listen` accept it only an object or could accept two args? Asking if this breaks current implementation and if this should be an overload method instead?

Reply:
The `listen` callback function is passed in `location` and `action` via an object. See https://github.com/remix-run/history/blob/main/docs/api-reference.md#history.listen